cpg_utils/workflows/metamist.py

```diff
@@ -384,11 +384,14 @@ def get_ped_entries(self, dataset: str | None = None) -> list[dict[str, str]]:
         """
         dataset = dataset or self.default_dataset
 
+        if get_config()['workflow']['access_level'] == 'test':
+            dataset += '-test'
+
         families = self.fapi.get_families(dataset)
         family_ids = [family['id'] for family in families]
         ped_entries = self.fapi.get_pedigree(
             internal_family_ids=family_ids,
-            response_type='json',
+            export_type='json',
```

Comment:
What's the difference between `response_type` and `export_type `?

Reply:
There might have been an update or something, but `response_type` isn't the valid param name https://sample-metadata.populationgenomics.org.au/documentation/FamilyApi.md#get_pedigree

Reply:
Thanks!